packages/cli/src/commands/generate/index.ts

```diff
@@ -15,14 +17,44 @@ export const generateSchema = async ({
     defaultSchema,
     schemas = [],
     tables = [],
+    schemaDefOverrides,
 }: GenerateSchemaOptions) => {
-    const result = await generate({
-        defaultSchema,
-        connectionString,
-        includeSchemas: schemas,
-        includeTables: tables,
-        outDir: out,
-    });
+    if (schemaDefOverrides === undefined) {
+        try {
+            return await generate({
+                defaultSchema,
+                connectionString,
+                includeSchemas: schemas,
+                includeTables: tables,
+                outDir: path.resolve(path.join(process.cwd(), out)),
+            });
+        } catch (error) {
+            console.log('Error: ', JSON.stringify(error, null, 2));
+        }
+    } else if (isValidSchemaDefOverrides(schemaDefOverrides)) {
+        try {
+            return await generate({
```

Comment:
you don't need to calls to `generate`, you should call it just once and always pass the overrides, which can possibly be empty.